src/Hive.IO/Hive.IO/Forms/BuildingInputState.cs

```diff
@@ -86,6 +86,8 @@ private void RaiseAllPropertiesChangedEvent()
                     "UValueOpaque",
                     "UValueTransparent",
                     "GValue",
+                    "GValueTotal",
+                    "ShadingSetpoint",
```

Comment:
@daren-thomas is this possibly redundant? I forgot to add `"ShadingSetpoint"` to this list before, but the green color font in the form upong changing the default value was already working.

Also, what is the difference to `var properties` in line 84 and the `RaisePropertyChangeEvent...` in lines 76-82?

Reply:
@christophwaibel not totally redundant. The changing of the colors when editing `ShadingSetpoint` happens because of a call to `RaisePropertyChangedEventEx()` at the bottom of that property. BUT, what happens when you change the SIA room type? Then you need a way to tell ALL the little boxes that they should re-evaluate. That's what's happening in `RaiseAllPropertiesChangedEvent()`.

The difference between the ones in 84-105 and those in 76-82 is that the bottom ones also raise the property changed event for the fontweight and color. they're the sia properties.